GVRf/Framework/jni/objects/hybrid_object_jni.cpp

```diff
@@ -28,7 +28,9 @@ Java_org_gearvrf_NativeHybridObject_delete(JNIEnv * env,
 JNIEXPORT void JNICALL
 Java_org_gearvrf_NativeHybridObject_delete(JNIEnv * env,
         jobject obj, jlong jhybrid_object) {
-    delete reinterpret_cast<HybridObject*>(jhybrid_object);
+    HybridObject* hybrid_object = reinterpret_cast<HybridObject*>(jhybrid_object);
+    delete hybrid_object;
+    hybrid_object = nullptr;
```

Comment:
@jason2kim 

I tried in S6 and Note4 (Snapdragon) and could not reproduce this crash. 
I'm still using Mobile SDK 0.6.0 if that matters.

Do you have my latests commit? Can you share more details on how you got this crash?